build/Program.cs

```diff
@@ -0,0 +1,122 @@
+using System;
+using static Bullseye.Targets;
+using static SimpleExec.Command;
+
+const string Build = "build";
+const string Test = "test";
+const string Angular = "angular";
+const string CSharp = "csharp";
+const string PackageClean = "package-clean";
+const string PackageAll = "package-all";
+const string PackageMonoRepo = "package-monorepo";
+const string Tools = "tools";
+const string Swagger = "swagger";
+
+const string OutputFolder = "_output";
+const string TargetFramework = "net6.0";
+var AllRuntimes = new[]
+{
+    "win-x64",
+    "win-x86",
+    "linux-x64",
+    "linux-arm",
+    "linux-arm64",
+    "linux-musl-x64",
+    "osx-x64"
+};
+var MonoRepoRuntimes = new[]
+{
+    "linux-x64",
+    "linux-arm",
+    "linux-arm64"
+};
+
+static void PackageRuntime(string framework, string runtime)
+{
+    var outputFolder = $"{OutputFolder}/{runtime}/Kavita";
+
+    Console.WriteLine($"Creating {runtime} Package for {framework}");
+
+    Run(
+        "dotnet",
+        $"publish -c Release --self-contained --runtime {runtime} -o \"{outputFolder}\" --framework {framework}",
+        "."
+    );
+
+    Console.WriteLine("Recopying wwwroot due to bug");
+    Run("cp", $"-R ./API/wwwroot/ {outputFolder}/wwwroot");
+
+    Console.WriteLine("Copying Install information");
+    Run("cp", $" INSTALL.txt {outputFolder}/README.txt");
+
+    Console.WriteLine("Copying LICENSE");
+    Run("cp", $" LICENSE {outputFolder}/LICENSE.txt");
+
+    Console.WriteLine("Renaming API -> Kavita");
+    if (runtime == "win-x64" || runtime == "win-x86")
+    {
+        Run("cp", $"{outputFolder}/API.exe {outputFolder}/Kavita.exe");
+    }
+    else
+    {
+        Run("mv", $"{outputFolder}/API {outputFolder}/Kavita");
+    }
+
+    Console.WriteLine("Copying appsettings.json");
+    Run("cp", $" API/config/appsettings.json {outputFolder}/config/appsettings.json");
```

Comment:
This step might not be needed anymore given your, what I believe is a fix, in the csproj to copy the appsettings.json over on build.

Reply:
Removed